opal/browser/audio/param_schedule.rb

```diff
@@ -0,0 +1,24 @@
+module Browser; module Audio
+
+  class ParamSchedule
+    include Native
+
+    alias_native :value,               :setValueAtTime
```

Comment:
Ah, I don't know the API, I just assumed it was a single value, what are the two values it takes?